.github/workflows/ci.yml

```
@@ -19,13 +19,11 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
           allow-prereleases: true
+      - name: Copy libmagic into magic dir
+        run: ${{ ( startsWith( matrix.os, 'windows' ) && 'bash add_libmagic.sh' ) || 'sudo -E bash add_libmagic.sh' }}
```

```suggestion
        run: ${{ (runner.os == 'Windows' && 'bash add_libmagic.sh')  || 'sudo -E bash add_libmagic.sh' }}
```
[GitHub Actions: Detecting the operating system](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#detecting-the-operating-system)